api/admin/problem_details.py

```diff
@@ -253,3 +253,24 @@
     title=_("Multiple search services"),
     detail=_("You tried to create a new search service, but a search service is already configured."),
 )
+
+MISSING_CUSTOM_LIST = pd(
+    "http://librarysimplified.org/terms/problem/missing-custom-list",
+    status_code=404,
+    title=_("Missing custom list"),
+    detail=_("The specified custom list doesn't exist."),
+)
+
+CANNOT_CHANGE_LIBRARY_FOR_CUSTOM_LIST = pd(
+    "http://librarysimplified.org/terms/problem/cannot-change-library-for-custom-list",
+    status_code=400,
+    title=_("Cannot change library for custom list"),
+    detail=_("A custom list's associated library cannot be changed once it is set.."),
+)
+
+CANNOT_ADD_CUSTOM_LIST_ENTRY_WITHOUT_LICENSEPOOL = pd(
+    "http://librarysimplified.org/terms/problem/cannot-add-entry-without-licensepool",
+    status_code=400,
+    title=_("Cannot add custom list entry without license pool."),
+    detail=_("You can't add an entry without a license pool to a custom list."),
+)
```

Comment:
Will you want to prevent the addition of a Work to a list if the library has no LicensePool for it? It's possible for a library to lose Works because it loses access to a collection, so we can't stop this from happening in all cases, but it might make sense to make the editor filter out attempts to add Works to a list when we know they won't show up.

Reply:
I think I should remove this restriction - editors will be adding to the list from search results, and works without a pool won't show up in the search results. If a work loses its pool after being added, this restriction could make it impossible to save the list until you remove that work. I think it's better to leave the work in the list and it will still show in the admin interface but it won't show up in lanes. And if the pool comes back, it can start showing up again.